docker-compose.local.yml

```
@@ -0,0 +1,56 @@
+      - ./src:/app/src
+      - ./api_server.py:/app/api_server.py
+      - ./downloads:/app/downloads
+    depends_on:
+      postgres:
+        condition: service_healthy
+      redis:
+        condition: service_healthy
+    command: python api_server.py
+
+volumes:
+  postgres_data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)